cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -1419,11 +1419,7 @@ def test_create_non_deprecated_problems(self):
         self.assertIn('Advanced', button_names)
         self.assertEqual(len(templates[0]['templates']), 3)
         template_display_names = [template['display_name'] for template in templates[0]['templates']]
-<<<<<<< HEAD
-        self.assertEqual(sorted(template_display_names), sorted(['Annotation', 'Open Response Assessment', 'Peer Grading Interface']))
```

Comment:
TODO: See if we still need the `sorted` calls. We originally added this because our components came in under a different ordering.